reactjs-films-homework/src/shared/helpers/calculateMovieDuration.ts

```diff
@@ -0,0 +1,5 @@
+export const calculateMovieDuration = (runtime: number) => {
+  const hours = Math.floor(runtime / 60);
+  const minutes = runtime % 60;
+  return `${hours}:${minutes}`
```

Comment:
in case minutes or hours lower than 10 you can use padStart() read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)